app/view/style/themes/index.js

```diff
@@ -0,0 +1,7 @@
+import light from './light';
+import dark from './dark';
+
+export default {
+  light,
+  dark,
+};
```

Comment:
By using Babel version 7.x we can use re-exporting like below:

```javascript
export { default as App } from './App.js';
```

But it is ok now, I just say that to keep this repo consistent.